solr/test-framework/src/java/org/apache/solr/handler/BackupRestoreUtils.java

```diff
@@ -109,7 +110,7 @@ public static void runReplicationHandlerCommand(
   }
 
   static void executeHttpRequest(String requestUrl) throws IOException {
-    URL url = new URL(requestUrl);
+    URL url = URI.create(requestUrl).toURL();
```

Comment:
This is causing `TestSolrCoreSnapshots.testBackupRestore` to fail on Windows machines.

```
java.lang.IllegalArgumentException: Illegal character in query at index 117: http://127.0.0.1:64027/solr/admin/cores?action=BACKUPCORE&core=SolrCoreSnapshots_shard1_replica_n1&name=u&location=C:\Users\jenkins\workspace\Solr-main-Windows\solr\core\build\tmp\tests-tmp\org.apache.solr.core.snapshots.TestSolrCoreSnapshots_9D4706448466E215-001\tempDir-002&incremental=false&commitName=ydfbp
	at __randomizedtesting.SeedInfo.seed([9D4706448466E215:2BFC06FAB6D52C5B]:0)
	at java.base/java.net.URI.create(URI.java:883)
	at org.apache.solr.handler.BackupRestoreUtils.executeHttpRequest(BackupRestoreUtils.java:113)
```